src/database.py

```diff
@@ -32,7 +32,12 @@
 from nowcasting_datamodel.save.update import N_GSP
 from sqlalchemy.orm.session import Session
 
-from pydantic_models import GSPYield, LocationWithGSPYields
+from pydantic_models import (
+    GSPGenerations,
```

Comment:
Could we maybe name this something more specific to what the return type is/looks like? Something like `GSPYieldsGroupByDatetime` or just `GSPYieldsByDatetime`

Reply:
yea good idea